### ISSUE TITLE
FoundryVTT becomes increasingly unresponsive when playing sounds sequentially with Maestro

### ISSUE THREAD

**Issue by mvanduijvenvoorde** [death-save/maestro#97]
**Describe the bug**
FoundryVTT becomes increasingly unresponsive when playing sounds/music in a sequential playlist and eventually hangs so much that everybody in the game has to restart to continue playing.

Atropos, the FoundryVTT developer, seems to think that this is also possibly related to the use of the Maestro module. That's why I also posted an issue here as well as on the FoundryVTT gitlab.

````
Failed to load resource: net::ERR_CACHE_OPERATION_NOT_SUPPORTED
````

See for all the information, screenshots, reproduceability -> FoundryVTT gitlab issue:
[Foundry Gitlab issue 3949](https://gitlab.com/foundrynet/foundryvtt/-/issues/3949)

**Foundry Setup (please complete the following information):**
 - OS: Windows
 - Browser: Native FoundryVTT & Chrome
 - Foundry Version: 0.7.7
 - Maestro Version: 0.7.2

**Additional context**

Possible related issues:
[Foundry Gitlab issue 3949](https://gitlab.com/foundrynet/foundryvtt/-/issues/3949)
[Foundry Gitlab issue 4147](https://gitlab.com/foundrynet/foundryvtt/-/issues/4147)
[Maestro Gitlab issue 92](https://github.com/death-save/maestro/issues/92)

**Comment by buttonpushertv** [comment]
I will add that I have seen that error on my RPi FVTT server running v0.7.7 & Maestro 0.7.2. I have also seen that with a playlist playing sequentially, when running the Response Time module, the reported ping will climb, over time - about 30 or 40 minutes, to 17,000/ms and FVTT becomes very sluggish. Often, the only way to recover is to refresh or close the Chrome window and reopen a new one. I can't be sure if I have seen the error whenever this occurs, though - usually because the tab with FVTT has become so bad that I can't even get the Console window open.

For me, the test has been to run FVTT with these (and only these) modules:
- Maestro
- Response Times
- Tidy UI Game Settings
- Find the Culprit!
(All are up to date). If I leave FVTT up and running for 30 minutes, it will slow down to the point of requiring a refresh or closing/reopening the window every time. It really becomes apparent when you have an active combat running in the tracker - that seems to make the problem worsen quicker. The ping will keep climbing, but directly pinging the server in a terminal window shows a 5/ms ping the whole time.

Don't know if that helps or not.

**Comment by eclarke12** [comment]
Fixed in #96
